src/utils/helpers/with-unique-prop/with-unique-prop.hoc.js

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import guid from '../guid';
 
-const withUniqueName = (WrappedComponent) => {
-  class WithUniqueName extends React.Component {
+const withUniqueProp = (WrappedComponent) => {
```

Comment:
shouldn't that be `withUniqueId`?

Reply:
`withUniqueProp` can be expanded in the future. At the moment it offers `id` and `label` `guid()`.

Reply:
what do you think about `withUniqueIdProps`? `withUniqueProp` is too general in my opinion.

Reply:
Resolved in 608fe850ca58c210e3e62d5af6e4c713dba037b9